ReactSkia/RSkThirdPartyFabricComponentsProvider.cpp

```diff
@@ -13,17 +13,22 @@ namespace react {
 RSkComponentProviderProtocol RSkThirdPartyFabricComponentsProvider(const char *componentName) {
   // This function and hash table must be generated by codegen with all the linked thirdparty RSkComponentProviderProtocol
   static std::unordered_map<std::string, RSkComponentProviderProtocol> fabricComponentsClassMap = {
+    {"RNSVGCircle",RSkComponentProviderRNSVGCircleCls},
```

Comment:
As per Discussion @munezbn , This table will be autogenerated in future. To avoid compilation issue in case thirdparty package ins no integrated. Lets not add these mapping in code base, and can be done at the time of using the App.